### DISCUSSION TITLE
master/slave and upgrade scenarios

### DISCUSSION THREAD

**Discussion by prateekjainaa** [arkmq-org/arkmq-org-broker-operator#585] [Q&A] [answered]
Hi All,
I am trying to create a cluster of activemq using this operator but dont see any documentation around creating master/slave configurations. Also, regarding upgrade/rollback scenarios are there any examples?

By looking at older threads, it seems like master/slave deployment is not supported by this operator. Let me know if I am  missing something here.

Regards,
Prateek

**Comment by brusdev** [comment, marked as answer]
@prateekjainaa you are right live/backup deployment is not supported by this operator because it doesn't match the Kubernetes philosophy. The operator relays on Kubernetes to restart a faulty broker instance and provides the [message migration feature](https://artemiscloud.io/docs/tutorials/scaleup_and_scaledown/) if you need to scale down your broker cluster.
